eZ/Publish/Core/settings/fieldtypes.yml

```diff
@@ -439,6 +439,7 @@ services:
     ezpublish.fieldType.ezuser:
         class: "%ezpublish.fieldType.ezuser.class%"
         parent: ezpublish.fieldType
+        arguments: ["@ezpublish.api.service.inner_user"]
```

Comment:
Is `@ezpublish.api.service.inner_user` instead of `@ezpublish.api.service.user` intentionally used?

Reply:
Good catch @adamwojs, I missed that :)

Reply:
Thanks, fixed